spec/helpers/time_helper_spec.rb

```diff
@@ -51,6 +51,15 @@
         expect(helper.set_date_max_to_plan_end_of_year(individual_enrollment)).to eq(latest_date)
       end
     end
+
+    context "for employee with cobra enrollment" do
+      it "sets the plan years last day on the calendar widget and allows to change the enrollment termination date" do
+        enrollment.update_attributes(kind: "employer_sponsored_cobra")
+        enrollment.effective_on = (TimeKeeper.date_of_record - 7.days)
```

Comment:
why do we need to set the effective_on `(TimeKeeper.date_of_record - 7.days)`?

Reply:
I just needed to assign an effective date earlier than today’s date, it doesn’t have to be exact. I could have fetched the effective date from the benefit package, but since the code is already merged and it's not critical. We are good to leave it as is for now.

The RSpec should verify that the enrollment’s set_date_max_to_plan_end_of_year matches enrollment.sponsored_benefit_package.end_on. Which seems to be working as expected.